libs/extensions/angular/image-banner/src/image-banner.component.ts

```diff
@@ -36,6 +36,11 @@ export class ImageBannerComponent {
    */
   @Input() externalLink: string | undefined;
 
+  /**
+   * The text of the dismiss button. Will be used as title for sighted users and aria-label for accessibility.
+   */
+  @Input() dismissButtonText: string | undefined;
```

Comment:
Consider renaming this `dismissButtonLabel` throughout, now that it is no longer a title.